File: Unidad_III/Brandon/Java/include/HolaMundo.java
public class HolaMundo 
{
	//Atributos
	String message;

	//Constructores
	public HolaMundo()
	{
		message = "Hola mundo!";
	}
	public HolaMundo(String mensaje)
	{
		message = mensaje;
	}

	//Métodos
	public void say_hello()
	{
		System.out.println(message);
	}

}

File: Unidad_III/Brandon/ejemplo_2/C++/HolaMundo/include/HolaMundo.cpp
#include <HolaMundo.hpp>

HolaMundo::HolaMundo( void ){}
HolaMundo::HolaMundo(std::string mensaje)
{
    message = mensaje;
}

void HolaMundo::say_hello(void)
{
    std::cout << message << std::endl;
}